Code/GraphMol/Depictor/RDDepictor.h

```diff
@@ -39,6 +39,21 @@ class RDKIT_DEPICTOR_EXPORT DepictException : public std::exception {
   std::string _msg;
 };
 
+//! \brief Set the path to the file containing the ring system templates
+/*!
+
+  \param template_path the file path to a file containing the ring system templates.
+      Each template must be a single line in the file represented using CXSMILES,
+      and the structure should be a single ring system.
+
+  \return true if the file was successfully loaded, false otherwise
+
+*/
+bool RDKIT_DEPICTOR_EXPORT setRingSystemTemplates(const std::string template_path);
```

Comment:
Not critical, but we normally don't use underscores:
```suggestion
bool RDKIT_DEPICTOR_EXPORT setRingSystemTemplates(const std::string templatePath);
```

Reply:
Please make this change for all the function arguments

Reply:
Thanks, I will update all of these!